configure.ac

```diff
@@ -387,16 +387,19 @@ AC_PROG_CC_C11
 if test "x$ac_cv_prog_cc_c99" = "xno"; then
    dnl we really should ERROR, lets see how far can come without
    AC_MSG_WARN([This compiler ${CC} has no c99 support])
+   INSERT_HAVE_C99="#undef HAVE_C99"
```

Comment:
HAVE_C99 is a little bit short and could also be used by something else, we should probably rename this to something else.

Reply:
Yes, probably should prepend our prefix to it.

Reply:
I renamed HAVE_C99 to SAFECLIB_HAVE_C99, this should not collide with other definitions.